Dockerfile.centos

```diff
@@ -0,0 +1,58 @@
+# Copyright (c) 2012-2016 Red Hat, Inc
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#   Dharmit Shah  - Initial implementation
+#   Mario Loriedo - Improvements
+#
+# To build it, run in the repository root:
+#  `docker build -t registry.centos.org/eclipse/che-server -f Dockerfile.centos .`
+#
+# To run it:
+#  docker run --net=host \
+#             --name che \
+#             -v /var/run/docker.sock:/var/run/docker.sock \
+#             -v /home/user/che/lib:/home/user/che/lib-copy \
+#             -v /home/user/che/workspaces:/home/user/che/workspaces \
+#             -v /home/user/che/storage:/home/user/che/storage \
+#             registry.centos.org/eclipse/che-server
+#           
+FROM registry.centos.org/centos/centos:latest
```

Comment:
@benoitf 

> but for users wanted to use this Dockerfile, they are not local to the pipeline

Agreed. Instead of using the Docker hub, it's pulling from registry.centos.org.

> Else it would act for me, like storing in eclipse/che repository a file that is internal to registry.centos.org and used privately

Nothing's getting used privately. Anyone can pull from registry.centos.org. Also, anyone can build the image using the exact same Dockerfile on their local system.

If that doesn't answer your question, I'm not sure I understand your concern here.